src/probnum/quad/bq_methods/stop_criteria/_stopping_criterion.py

```diff
@@ -0,0 +1,101 @@
+"""Stopping criteria for Bayesian quadrature."""
+
+from typing import Callable
+
+import numpy as np
+
+from probnum.quad.bq_methods.bq_state import BQState
+from probnum.randvars import Normal
+from probnum.type import FloatArgType, IntArgType
+
+
+class StoppingCriterion:
+    """Base class for a stopping criterion.
+
+    Parameters
+    ----------
+    stopping_criterion :
+        A function that determines whether a convergence criterion is reached.
+    """
+
+    def __init__(self, stopping_criterion: Callable):
+        self._stopping_criterion = stopping_criterion
+
+    def __call__(self, integral_belief: Normal, bq_state: BQState) -> bool:
+        """Evaluate the stopping criterion.
+
+        Parameters
+        ----------
+        integral_belief :
+            Current Gaussian belief about the integral.
+        bq_state:
+            State of the BQ loop.
+
+        Returns
+        -------
+        has_converged:
+            Boolean whether a stopping criterion has been reached
+        """
+        return self._stopping_criterion(bq_state)
+
+
+class IntegralVariance(StoppingCriterion):
```

Comment:
Let;s make it more clear that this is a stopping criterion and not the integral variance, e.g., `IntegralVarianceThreshold`. Otherwise the namespace starts to be confusing.